.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Join The Community
+    url: https://www.kubeflow.org/docs/about/community/
+    about: Join our slack channel and access our meeting schedule.
```

Comment:
join our slack channel `wg-manifests` here https://www.kubeflow.org/docs/about/community/. This link also contains our meeting schedule.